gtsam/hybrid/HybridGaussianFactorGraph.cpp

```diff
@@ -233,6 +233,26 @@ continuousElimination(const HybridGaussianFactorGraph &factors,
   return {std::make_shared<HybridConditional>(result.first), result.second};
 }
 
+/* ************************************************************************ */
+/**
+ * @brief Exponential log-probabilities after performing
+ * the necessary normalizations.
+ *
+ * @param logProbabilities DecisionTree of log-probabilities.
+ * @return AlgebraicDecisionTree<Key>
+ */
+static AlgebraicDecisionTree<Key> exponentiateLogProbabilities(
```

Comment:
Consider renaming to probabilitiesFromLogValues and replace “necessary normalization” in comment with: “Exponentiate log-values, not necessarily normalized, normalize, and return as ADT.”